app/assets/stylesheets/components/_login-banner.scss

```diff
@@ -0,0 +1,36 @@
+.login-banner {
+  background-color: $color-gray-cool-light;
+  color: $color-primary;
+  padding: 1rem 0;
+}
+
+.homepage-login-banner .login-banner {
+  @include media($nav-width) {
+    background-color: $color-primary;
+    color: $color-white;
+    font-size: $h3-font-size;
+    padding: 2rem 0;
+
+    a {
+      color: $color-white;
+    }
+  }
+}
+
+.login-banner-text {
+  @include outer-container();
+  @include padding(null $site-margins-mobile);
+  font-size: $small-font-size;
+  text-align: center;
+
+  @include media($medium-screen) {
+    font-size: $base-font-size;
+  }
+
+  @include media($nav-width) {
+    @include padding(null $site-margins);
+    text-align: left;
+  }
+
+  max-width: $site-max-width;
```

Comment:
This property and value belongs between lines 23 and 24 with the rest of the styles declared on `.login-banner-text`.